ONSdigital/ras-rm-auth-service
# RAS-933 Auth SQL logs print email

# What and why?
This PR is to stop auth from logging the user's email when a database error occurs. The tokens method was missing a try catch compared to the rest of the repo and it would have been too much effort to update the other endpoint so we could have the database error handling only in the decorator.
# How to test?
- deploy this pr to your environment
- ensure login work normally for rops and fronstage
- singout of frontstage
- run `kubectl delete statefulsets.app postgres`
- try logging into frontstage
- check the auth logs and make sure there is no database query or multi-line stacktrace

Note: the last three steps can be done before deploying as well to see the original state of the logs, but make sure to redeploy postgres and restart your deployments to reconnect to the database
# Jira
https://jira.ons.gov.uk/browse/RAS-933
